superset/explore/form_data/commands/create.py

```diff
@@ -49,6 +53,7 @@ def run(self) -> str:
                     "form_data": form_data,
                 }
                 cache_manager.explore_form_data_cache.set(key, state)
+                cache_manager.explore_form_data_cache.set(contextual_key, key)
             return key
         except SQLAlchemyError as ex:
```

Comment:
There isn't SQLAlchemy operation, please change to a more explicit exception.

Reply:
The `check_access` function can emit SQLAlchemy errors from different functions. I'm just catching here to unify the exception type.

Reply:
okay. got it.